test_scripts/Defects/5_0/2454_SDL_does_not_clear_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2454
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
```

Comment:
according to defect description there is no any registered apps. please add in description steps with apps registration for adding data in app_info.dat

Reply:
@GetmanetsIrina Added in bf1c6b75fe5f4e1f3d891034a6faf412bcf250e1.